src/05-derivatives/04-functional-derivatives.md

```diff
@@ -47,51 +47,65 @@ The following metadata JSON fields are valid for derivative maps:
 | Threshold      | String describing threshold used for determining graph edges.             | `dcb`, `dcw`, `ecb`, `ecw` |
 | Method         | String describing method used to calculate measure.                       | `dcb`, `dcw`, `ecb`, `ecw` |
 
-## Time series and regressors
+## Time series
 
-For the purposes of this section, a time series or regressor is a numeric series
-of values corresponding to volumes in a BOLD series. Time series is specifically
-used to indicate an aggregate time series within an ROI, while regressor is used
-as a more general term indicating series that may be derived from other sources,
-such as head motion estimates or physiological recordings.
+A time series is an ordered series of numeric values in chronological order.
+For the purposes of this section, a "regressor" is a time series with one entry
+per volume in a BOLD series.
```

Comment:
Hmm. I think I don't use "regressor" as much as I thought I would in the remainder of the section. It does get used it in a couple places, but it's not clear that a specific definition is required. Or whether the section wouldn't be improved by purging "regressor" altogether.

Reply:
+1 for purging regressor—I don't think the notion is doing any work at this point.